keep/api/api.py

```diff
@@ -263,6 +264,8 @@ async def on_startup():
             logger.info("Providers loaded successfully")
             WorkflowStore.provision_workflows_from_directory(SINGLE_TENANT_UUID)
             logger.info("Workflows provisioned successfully")
+            provision_dashboards(SINGLE_TENANT_UUID)
```

Comment:
on workflows, removing workflow from the env var will de-provision it from Keep.

Reply:
can dashboard be de-provisioned?

Reply:
IDK. I guess we can do it if the user asks for it. For now, no.